src/test/java/com/wikia/webdriver/common/core/api/CuratedContent.java

```diff
@@ -0,0 +1,39 @@
+package com.wikia.webdriver.common.core.api;
+
+import com.wikia.webdriver.common.core.annotations.User;
+import com.wikia.webdriver.common.core.configuration.Configuration;
+import com.wikia.webdriver.common.core.url.UrlBuilder;
+
+import org.apache.http.message.BasicNameValuePair;
+
+import java.util.ArrayList;
+
+public class CuratedContent extends ApiCall {
+
+  public CuratedContent() {
+  }
+
+  @Override protected String getURL() {
+    return new UrlBuilder().getUrlForWiki(Configuration.getWikiName())
+           + "wikia.php?controller=CuratedContent&method=setData";
+  }
+
+  @Override protected User getUser() {
+     return User.STAFF;
+  }
+
+  @Override
+  protected ArrayList<BasicNameValuePair> getParams() {
+    return null;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Return an empty collection instead of null. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1168)